### ISSUE TITLE
May be a bug in APFG_model.py

### ISSUE THREAD

**Issue by Nora-Zhang98** [wanglei0618/A-PFG#1]
Dear author,
Thanks for your excellent work! When I try to reimplement your work, I find a discrepancy with your paper.
The code line is https://github.com/wanglei0618/A-PFG/blob/8ba11eb036bca124c5c678a9030b043bb4df4787/maskrcnn_benchmark/APFG/APFG_model.py#L187

Accoarding the Eq(10) in your paper, maybe the
L_DA = torch.mean((torch.linalg.norm((mu_semantic - log_var_semantic), dim=1)**2
should be replaced by
L_DA = torch.mean((torch.linalg.norm((mu_semantic - mu_visual), dim=1)**2

**Comment by wanglei0618** [comment]
Thanks very much for your feedback. This indeed is a bug, and we have fixed it.
